--- otp-OTP-21.3.8.6/erts/configure.ac.orig	2019-07-11 17:18:55.000000000 +0200
+++ otp-OTP-21.3.8.6/erts/configure.ac	2019-09-09 17:44:05.368217526 +0200
@@ -459,7 +459,8 @@ if test -z "$XSLTPROC"; then
   AC_MSG_WARN([No 'xsltproc' command found: the documentation cannot be built])
 fi
 
-AC_CHECK_PROGS(FOP, fop)
+dnl AC_CHECK_PROGS(FOP, fop)
+AC_SUBST(FOP)
 if test -z "$FOP"; then
   FOP="$ERL_TOP/make/fakefop"
   echo "fop" >> doc/CONF_INFO

