diff -dur otp_src_R9C-2.orig/lib/crypto/c_src/Makefile.in otp_src_R9C-2/lib/crypto/c_src/Makefile.in
--- otp_src_R9C-2.orig/lib/crypto/c_src/Makefile.in	2003-07-07 14:32:05.000000000 +0200
+++ otp_src_R9C-2/lib/crypto/c_src/Makefile.in	2004-09-01 13:03:20.908573964 +0200
@@ -56,7 +56,7 @@
 TYPE_FLAGS = -O2
 endif
 
-ALL_CFLAGS = @CFLAGS@ @DEFS@ $(TYPE_FLAGS) $(INCLUDES)
+ALL_CFLAGS = @CFLAGS@ @DEFS@ @DED_CFLAGS@ $(TYPE_FLAGS) $(INCLUDES)
 
 # ----------------------------------------------------
 # Release directory specification
diff -dur otp_src_R9C-2.orig/lib/ic/c_src/Makefile.in otp_src_R9C-2/lib/ic/c_src/Makefile.in
--- otp_src_R9C-2.orig/lib/ic/c_src/Makefile.in	2004-04-08 14:44:14.000000000 +0200
+++ otp_src_R9C-2/lib/ic/c_src/Makefile.in	2004-09-01 13:04:34.243490871 +0200
@@ -110,7 +110,7 @@
 
 OBJ_FILES= $(C_FILES:%.c=$(OBJDIR)/%.o) 
 
-ALL_CFLAGS = @CFLAGS@ @DEFS@ -I$(INCDIR) $(ERL_INTERFACE_FLAGS) $(CFLAGS) 
+ALL_CFLAGS = @CFLAGS@ @DEFS@ @DED_CFLAGS@ -I$(INCDIR) $(ERL_INTERFACE_FLAGS) $(CFLAGS) 
 
 # ----------------------------------------------------
 # Targets
Only in otp_src_R9C-2/lib/ic/c_src: Makefile.in~
diff -dur otp_src_R9C-2.orig/lib/odbc/c_src/Makefile.in otp_src_R9C-2/lib/odbc/c_src/Makefile.in
--- otp_src_R9C-2.orig/lib/odbc/c_src/Makefile.in	2004-05-25 19:37:40.000000000 +0200
+++ otp_src_R9C-2/lib/odbc/c_src/Makefile.in	2004-09-01 13:05:27.679036798 +0200
@@ -83,7 +83,7 @@
 
 EIINCLUDE = $(EIROOT)/include
 INCLUDES = -I. -I$(EIINCLUDE) -I$(ODBCINCLUDE)
-CFLAGS = $(subst -O2,-g,@CFLAGS32@)
+CFLAGS = $(subst -O2,-g,@CFLAGS32@) @DED_CFLAGS@
 #CFLAGS = @CFLAGS32@
 LDFLAGS += -g
 CC = @CC32@
Only in otp_src_R9C-2/lib/odbc/c_src: Makefile.in~
diff -dur otp_src_R9C-2.orig/lib/orber/c_src/Makefile.in otp_src_R9C-2/lib/orber/c_src/Makefile.in
--- otp_src_R9C-2.orig/lib/orber/c_src/Makefile.in	2003-07-22 15:27:15.000000000 +0200
+++ otp_src_R9C-2/lib/orber/c_src/Makefile.in	2004-09-01 13:06:07.082853761 +0200
@@ -64,7 +64,7 @@
 TARGET_FILES= $(PROGRAMS:%=$(BINDIR)/%)
 OBJ_FILES= $(C_FILES:%.c=$(OBJDIR)/%.o) 
 
-ALL_CFLAGS = @CFLAGS@ @DEFS@ $(CFLAGS) 
+ALL_CFLAGS = @CFLAGS@ @DED_CFLAGS@ @DEFS@ $(CFLAGS) 
 
 # ----------------------------------------------------
 # Targets
Only in otp_src_R9C-2/lib/orber/c_src: Makefile.in~
diff -dur otp_src_R9C-2.orig/lib/os_mon/c_src/Makefile.in otp_src_R9C-2/lib/os_mon/c_src/Makefile.in
--- otp_src_R9C-2.orig/lib/os_mon/c_src/Makefile.in	2003-07-07 14:05:00.000000000 +0200
+++ otp_src_R9C-2/lib/os_mon/c_src/Makefile.in	2004-09-01 13:06:38.578524410 +0200
@@ -68,7 +68,7 @@
 
 TARGET_FILES= $(PROGRAMS:%=$(BINDIR)/%)
 
-ALL_CFLAGS = @CFLAGS@ @DEFS@ $(CFLAGS)
+ALL_CFLAGS = @CFLAGS@ @DEFS@ @DED_CFLAGS@ $(CFLAGS)
 
 # ----------------------------------------------------
 # Targets
Only in otp_src_R9C-2/lib/os_mon/c_src: Makefile.in~
diff -dur otp_src_R9C-2.orig/lib/ssl/c_src/Makefile.in otp_src_R9C-2/lib/ssl/c_src/Makefile.in
--- otp_src_R9C-2.orig/lib/ssl/c_src/Makefile.in	2004-04-22 13:14:28.000000000 +0200
+++ otp_src_R9C-2/lib/ssl/c_src/Makefile.in	2004-09-01 13:07:22.272353241 +0200
@@ -44,7 +44,7 @@
 # Includes and libs
 # ----------------------------------------------------
 
-ALL_CFLAGS = @WFLAGS@ @CFLAGS@ @DEFS@ $(TYPE_FLAGS)
+ALL_CFLAGS = @WFLAGS@ @DED_CFLAGS@ @CFLAGS@ @DEFS@ $(TYPE_FLAGS)
 TARGET = @host@
 
 ifeq ($(TYPE),debug)
Only in otp_src_R9C-2/lib/ssl/c_src: Makefile.in~
